addons/website/static/src/scss/primary_variables.scss

```diff
@@ -1970,6 +1970,8 @@ $o-base-website-values-palette: (
         6: 2rem, // We don't use it.
     ),
 
+    'small-font-size': 0.875rem,
```

Comment:
Same comment: should not be forced in the base palette I think? Default to the bootstrap value?